### ISSUE TITLE
Error when loading homepage on smartedit

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#2784]
**Environment Details**
Spartacus: latest develop
Browser: Chrome 73.0.3683.86
OS: Mac Os
https://dev-com-2.accdemo.b2c.ydev.hybris.com:9002/smartedit/#!/storefront

**Steps to Reproduce**
In smartedit, go to homepage

**Actual Result**
This error is displayed in the console
![SEerror](https://user-images.githubusercontent.com/45409515/58654185-5499d480-82e5-11e9-89d9-7c70a61faec9.png)

**Comment by WeizhengSap** [comment]
Hi, if you see the error message `disallowed storefront...`, that means your storefront is not in the whitelist. Now, in our sample data, the default SPA url is `https://localhost:4200`. It uses `https` instead of `http`. So, you have to start SPA with `--ssl`.

**Comment by KateChuen** [comment]
I still get the disallowed error.
I'm starting SPA with --ssl: 
![SpaSSL](https://user-images.githubusercontent.com/45409515/58715148-40f97700-8394-11e9-8d60-fd4a1a84dd50.png)

In backoffice, I have it set to use: https://dev-com-2.accdemo.b2c.ydev.hybris.com:4200

But still getting the error:
![disallowedError](https://user-images.githubusercontent.com/45409515/58715290-8453e580-8394-11e9-9727-4a57e784e89e.png)

**Comment by KateChuen** [comment]
Closing, not an issue after changing the whitelist info.

**Comment by xavileci** [comment]
Hello.
I've some issue similar than this.
![image](https://user-images.githubusercontent.com/42523278/97885372-907d6880-1d27-11eb-800e-d5f7f1acdece.png)

I have an Apache Proxy Server configured with differents domains depending if I would like to access to storefront, smartedit, or backoffice site.
I've added below the header to permit cross iframe:
`	Header set Content-Security-Policy "frame-ancestors 'self' https://*.domain.com;"
`
From hac I've added whiteListedStorefronts property.
`INSERT_UPDATE SmartEditConfiguration;key[unique=true];value
;whiteListedStorefronts;"[""*.domain.com""]"
`
Any idea about what could be happening?
